spec/classes/config_spec.rb

```diff
@@ -263,6 +263,17 @@
           it { is_expected.not_to contain_mesos__property('marathon_zk') }
         end
       end
+
+      context 'configure marathon\'s logging levels' do
+        let(:params) {{
+            :log_levels => { 'mesosphere.chaos.http' => 'ERROR'}
+          }}
```

Comment:
Sorry, our Rubocop rules are fairly strict :sweat_smile: 
You should do:
```ruby
let(:params) { {:log_levels => {'mesosphere.chaos.http' => 'ERROR'}} }
```
or...
```ruby
let(:params) do
  {:log_levels => {'mesosphere.chaos.http' => 'ERROR'}}
end
```

Spaces around curly braces for do/end, and no spaces for hashes (in order to differentiate between the two).